cmd/siad/server.go

```diff
@@ -145,6 +151,12 @@ func (rs byVersion) Less(i, j int) bool {
 	return build.VersionCmp(rs[i].version(), rs[j].version()) >= 0
 }
 
+// Close implements the Close method for moduleCloser.
+func (mc *moduleCloser) Close() error {
+	fmt.Printf("Closing %v...\n", mc.name)
+	return mc.closer.Close()
+}
```

Comment:
hmm. Having in the `Printf` inside the `Close` feels wrong. I'd keep it outside.

Reply:
specifically, I'd embed the `io.Closer` in `moduleCloser`, so you don't need to define a new method at all

Reply:
Ah, gotcha. I'll push another commit